one.html

```diff
@@ -0,0 +1,42 @@
+<html>
+
+<script>
+
+num = prompt("Enter list of numbers");
+
+mainarr = num.split(',');
+
+var arr=[], freq=[];

```

Comment:
Prefer `let` over `var` and `const` over `let` if we are not modifying variable value further